pioneer/packages/joy-proposals/src/Proposal/ProposalPreviewList.tsx

```diff
@@ -50,13 +51,34 @@ type ProposalPreviewListProps = {
   bestNumber?: BlockNumber;
 };
 
+const ProposalFilterCountBadge = styled.span`
+  background-color: rgba(0, 0, 0, .3);
+  color: #fff;
+
+  border-radius: 10px;
+  height: 19px;
+  min-width: 19px;
+  padding: 0 4px;
+
+  font-size: .8rem;
+  font-weight: 500;
+  line-height: 1;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  margin-left: 6px;
+`;
+
 function ProposalPreviewList ({ bestNumber }: ProposalPreviewListProps) {
   const transport = useTransport();
   const [proposals, error, loading] = usePromise<ParsedProposal[]>(() => transport.proposals.proposals(), []);
   const [activeFilter, setActiveFilter] = useState<ProposalFilter>('All');
 
   const proposalsMap = mapFromProposals(proposals);
   const filteredProposals = proposalsMap.get(activeFilter) as ParsedProposal[];
+  const sortedProposals = filteredProposals.sort((p1, p2) => p2.id.cmp(p1.id));
```

Comment:
Might be worth to move this to `mapFromProposals` so we do it only once and have them sorted by newest automatically.

Reply:
It won't change the number of sorts though, `mapFromProposals` is called on every render, same as `sort` in the code as it is. With the current approach, we may actually do less sorting because we only sort the proposals for the currently selected filter, in `mapFromProposals` we would sort all the proposals. That's a micro-optimization though 😄

Reply:
If you think that's better for readability though, let me know and I will change it.

Reply:
My bad! In terms of readability, your version is more explicit, so let's keep it that way

Reply:
:+1: